scripts/check-release-contracts.mjs

```
@@ -0,0 +1,68 @@
+assert.match(publishWorkflow, /npm view .*version >"\$view_output" 2>&1/);
+assert.match(publishWorkflow, /grep -Eq .*E404\|404/);
+assert.match(publishWorkflow, /exit "\$view_status"/);
+assert.match(publishWorkflow, /\[\[ "\$package_version" == \*-\* \]\]/);
+assert.match(publishWorkflow, /dist_tag=next/);
+assert.match(publishWorkflow, /npm publish .*--provenance.*--tag "\$dist_tag"/);
+assert.match(publishWorkflow, /Skipping already published/);
+assert.match(menuWorkflow, /tags: \["menu-bar-v\*"\]/);
+assert.match(menuWorkflow, /--target universal-apple-darwin/);
+assert.match(menuWorkflow, /environment: publish/);
+
+console.log("Release boundaries and channels satisfy the documented contracts.");
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: clear, info.

(no-console)